exercises/E-strings-concatenation/exercise.js

```diff
@@ -1,3 +1,6 @@
 // Start by creating a variable `message`
+var greetingStart = "Hi guys my name is ";
+let fname = "Danti";
```

Comment:
I see everything is fine, but remember to use camelCase when naming your variables.
let fname > **let firstName or fName**